### ISSUE TITLE
Keyboard trap when cancelling Print

### ISSUE THREAD

**Issue by jphinspace** [highcharts/highcharts#21880]
#### Expected behaviour
With the accessibility and export modules added, and after changing my mind about printing, I want to still be able to use the tab button to navigate in and through the focusable elements on the webpage containing a chart.

#### Actual behaviour
Tab navigation is stuck in a loop within the chart after cancelling out of the Print popup. Shift+Tab to go backwards still works.

#### Live demo with steps to reproduce
https://jsfiddle.net/o3Lzgf9x
Click or navigate to the title text (tabindex is "0", it should be selectable).
Hit tab a few times to select the hamburger button.
Select "Print Chart"
Select "Cancel" in the print popup
Tab navigation now is trapped in the chart.
Escape is only possible using Shift+Tab to go backwards.

#### Product version
HighCharts 11.4.8

#### Affected browser(s)
Chrome
Brave

**Comment by jphinspace** [comment]
Confirmed issue still present in 11.4.8

Found similar-looking issue https://github.com/highcharts/highcharts/issues/21554 but it looks like the fix doesn't resolve this case.

**Comment by marvin19** [comment]
Hi @jphinspace,

It looks like you have modified the Print-function. If I run `chart.afterPrint()` in the `setTimeout()` instead of `origParent.appendChild(container);` , I am able to focus on the div with the `tabindex` again and I am [not trapped in the chart](https://jsfiddle.net/maritavindedal/80d5u4a3/). 

If I [remove the Print-function](https://jsfiddle.net/maritavindedal/64obcw7s/), the problem also seems to be gone.
